docs/intro/quickstart/index.rst

```diff
@@ -0,0 +1,59 @@
+.. _ref_quickstart:
+
+==========
+Quickstart
+==========
+
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+  setup
+  modeling
+  working
+  workflow
+  access
+  inheritance
+  dynamic
+
+
+Welcome to our quickstart tutorial! Together, we'll create a simple HTTP API for a Flashcards application using Next.js and Gel. This practical project will let users build and manage their own study decks, with each flashcard featuring customizable text on both sides - making it perfect for studying, memorization practice, or creating educational games.
+
+Don't worry if you're new to Gel - we'll have you up and running with a Next.js starter project and a local Gel database in just about 5 minutes. From there, we'll guide you through building the complete application in roughly 60-90 minutes.
+
+Our Flashcards app will be a modern web application with the following features:
+
+* Create, edit and delete flashcard decks
+* Add and remove cards from decks
+* Display cards with front/back text content
+* Simple UI with Next.js and Tailwind CSS
+* Clean, type-safe data modeling using Gel's schema system
```

Comment:
How about changing this to "By the end of this tutorial, you will:
- Learn how to create, update and delete Flashcards with Gel 
- Learn how to select data from Gel and display in the UI
- Create a simple UI with Next.js and Tailwind CSS
- Explore type-safe data modeling using Gel's schema system"
? So that it focuses more on learning Gel while doing something else rather than on the app.

Reply:
Yeah, I like that! It's a strict improvement no matter what we decide about the rest of the content. FWIW, the third bullet ("Create a simpleUI with Next.js and Tailwind CSS") might be something we decide to drop by making the starting point a fully functional app with static, mock data, and we spend the whole tutorial working through replacing this static data layer with a Gel-powered one. I'd love to here your thoughts on that over on [the main thread](https://github.com/edgedb/edgedb/pull/8269#discussion_r1947387337) ☝️